libs/libsodium/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2009-2016 OpenWrt.org
+# Copyright (C) 2009-2017 OpenWrt.org
```

Comment:
One or two years ago, someone from OpenWrt asked me to update this when I did a PR. So this time I wanted to be one move ahead. Is it not sufficient to be package maintainer and contributor ?